packages/design-system-web/src/components/geov-map-places/geov-map-places.tsx

```diff
@@ -0,0 +1,293 @@
+          bind( replace( str(?rep), "^[^0-9\.-]*([-]?[0-9\.]+) .*$", "$1" ) as ?long )
+          bind( replace( str(?rep), "^.* ([-]?[0-9\.]+)[^0-9\.]*$", "$1" ) as ?lat )
+          ${
+            this.queryBoundingBox
+              ? `
+          FILTER (
+            xsd:double(?lat) >= ${bounds._sw.lat.toFixed(3)} &&
+            xsd:double(?lat) <= ${bounds._ne.lat.toFixed(3)} &&
+            xsd:double(?long) >= ${bounds._sw.lng.toFixed(3)} &&
+            xsd:double(?long) <= ${bounds._ne.lng.toFixed(3)}
+        )`
+              : ''
+          }
+        }
+        LIMIT ${this.limit}
+          `;
+
+      map.on('load', () => {
+        map.addSource('places', {
+          type: 'geojson',
+          data: this.markers,
+          cluster: true,
+          clusterMaxZoom: 12, // Max zoom to cluster points on
+          clusterRadius: 40, // Radius of each cluster when clustering points (defaults to 50)
+        });
+        const computedStyle = getComputedStyle(this.el);
+        const customColors = [
+          computedStyle.getPropertyValue('--circle-color-small') || 'red',
+          computedStyle.getPropertyValue('--circle-color-medium') || 'green',
+          computedStyle.getPropertyValue('--circle-color-large') || 'grey',
+        ];
+
+        map.addLayer({
+          id: 'clusters',
+          type: 'circle',
+          source: 'places',
+          filter: ['has', 'point_count'],
+          paint: {
+            // Use step expressions (https://maplibre.org/maplibre-style-spec/#expressions-step)
+            // with three steps to implement three types of circles:
+            //   * --circle-color-small, 20px circles when point count is less than 100
+            //   * --circle-color-medium, 30px circles when point count is between 100 and 750
+            //   * --circle-color-large, 40px circles when point count is greater than or equal to 750
+            'circle-color': ['step', ['get', 'point_count'], customColors[0], 100, customColors[1], 750, customColors[2]],
+            'circle-radius': ['step', ['get', 'point_count'], 20, 100, 30, 750, 40],
+          },
+        });
+
+        map.addLayer({
+          id: 'cluster-count',
+          type: 'symbol',
+          source: 'places',
+          filter: ['has', 'point_count'],
+          layout: {
+            'text-field': '{point_count_abbreviated}',
+            'text-size': 12,
+          },
+        });
+
+        //   Add popups to the markers
+        const handleMarkerClick = e => {
+          const coordinates = e.features[0].geometry.coordinates;
+          const description = e.features[0].properties.name;
+          const link = this.projectID ? `${e.features[0].properties.link}?p=${this.projectID}` : e.features[0].properties.link;
+          new Popup().setLngLat(coordinates).setHTML(`<div><a href = "${link}" target="_blank">${description}</a></div>`).addTo(map);
+        };
```

Comment:
Use MapLibre imported by CDN to avoid bundeling MapLibre into the webcomponent.
```suggestion
          new MapLibre.Popup().setLngLat(coordinates).setHTML(`<div><a href = "${link}" target="_blank">${description}</a></div>`).addTo(map);
```